react/BarContextProvider/index.spec.jsx

```diff
@@ -172,27 +172,4 @@ describe('BarContextProvider', () => {
 
     expect(queryByText(mockVoidWebviewService)).toBeInTheDocument()
   })
-
-  it('should not try to provide a cozy-intent context if one is provided', () => {
-    // Set Web context
```

Comment:
why did we delete this test?

Reply:
It was a faulty test, updating cozy-intent revealed that. Basically this test is wrong and expects the wrong result (another test expects the actual result in this file)